testapp/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "testapp",
   "version": "0.0.1",
   "scripts": {
-    "start": "concurrently \"npm run tsc:w\" \"npm run lite\" ",
-    "tsc": "tsc",
-    "tsc:w": "tsc -w",
-    "lite": "lite-server"
+    "install_testapp": "npm install && npm run tsc",
```

Comment:
This should probably be `postinstall`.  Obviously we're installing the testapp, since we're in that folder.  And `postinstall` isn't a problem since we never publish the testapp.